### DISCUSSION TITLE
[v5] model preload relations from auth

### DISCUSSION THREAD

**Discussion by gabbmariz** [adonisjs/core#2982] [Help] [answered]
I have two models "User" and "Profile", I thought of making a middleware so that the data of "Profile" is saved in the authentication along with the "User" but I have no idea how to do this, would anyone help?

**obs:** auth.user.profile return undefined

**Comment by capopovici** [comment, marked as answer]
maybe try `auth.user.load('profile')` ?

**Reply by gabbmariz** [reply]
I tested the auth.user.load('profile') on the control and it doesn't work, so I created a middleware and made it run the 'auth.user.load('profile') and it worked!
